### PR TITLE
Release v0.6.22

### PR DESCRIPTION
**Pull request by pfmooney**
Release v0.6.22 in the v0.6.x branch so illumos consumers can enjoy the benefits of #1294 

This PR also fixes up the URL listings in `Cargo.toml` and brings the file include/exclude rules in line with #1311.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,20 +6,22 @@ name          = "mio"
...
-exclude       = [
-  ".gitignore",
-  ".travis.yml",
-  "deploy.sh",
+include       = [
+  "Cargo.toml",
+  "LICENSE",
+  "README.md",
+  "CHANGELOG.md",
+  "src/**/*.rs"
```
Comment (Thomasdezeeuw) [inline Cargo.toml:24]: I'm pretty sure these have to be "/path", I ran into issues previously without "/", but those might have been fixed.

Comment (pfmooney) [reply Cargo.toml:24]: Seems to work for `cargo package`:
```
[pmooney@host mio]$ git log -n1 --oneline
4555534 (HEAD -> release-0.6.22, origin/release-0.6.22, v0.6.x) Release v0.6.22
[pmooney@italica mio]$ cargo package
   Packaging mio v0.6.22 (/home/pmooney/git/rust/mio)
   Verifying mio v0.6.22 (/home/pmooney/git/rust/mio)
    Updating crates.io index
   Compiling libc v0.2.69
   Compiling log v0.4.8
   Compiling cfg-if v0.1.10
   Compiling slab v0.4.2
   Compiling iovec v0.1.4
   Compiling net2 v0.2.33
   Compiling mio v0.6.22 (/home/pmooney/git/rust/mio/target/package/mio-0.6.22)
    Finished dev [unoptimized + debuginfo] target(s) in 2.77s
[pmooney@host mio]$ tar tzf target/package/mio-0.6.22.crate
mio-0.6.22/CHANGELOG.md
mio-0.6.22/Cargo.toml.orig
mio-0.6.22/Cargo.toml
mio-0.6.22/LICENSE
mio-0.6.22/README.md
mio-0.6.22/src/channel.rs
mio-0.6.22/src/deprecated/event_loop.rs
mio-0.6.22/src/deprecated/handler.rs
mio-0.6.22/src/deprecated/io.rs
mio-0.6.22/src/deprecated/mod.rs
mio-0.6.22/src/deprecated/notify.rs
mio-0.6.22/src/deprecated/unix.rs
mio-0.6.22/src/event_imp.rs
mio-0.6.22/src/io.rs
mio-0.6.22/src/lazycell.rs
mio-0.6.22/src/lib.rs
mio-0.6.22/src/net/mod.rs
mio-0.6.22/src/net/tcp.rs
mio-0.6.22/src/net/udp.rs
mio-0.6.22/src/poll.rs
mio-0.6.22/src/sys/fuchsia/awakener.rs
mio-0.6.22/src/sys/fuchsia/eventedfd.rs
mio-0.6.22/src/sys/fuchsia/handles.rs
mio-0.6.22/src/sys/fuchsia/mod.rs
mio-0.6.22/src/sys/fuchsia/net.rs
mio-0.6.22/src/sys/fuchsia/ready.rs
mio-0.6.22/src/sys/fuchsia/selector.rs
mio-0.6.22/src/sys/mod.rs
mio-0.6.22/src/sys/unix/awakener.rs
mio-0.6.22/src/sys/unix/dlsym.rs
mio-0.6.22/src/sys/unix/epoll.rs
mio-0.6.22/src/sys/unix/eventedfd.rs
mio-0.6.22/src/sys/unix/io.rs
mio-0.6.22/src/sys/unix/kqueue.rs
mio-0.6.22/src/sys/unix/mod.rs
mio-0.6.22/src/sys/unix/ready.rs
mio-0.6.22/src/sys/unix/tcp.rs
mio-0.6.22/src/sys/unix/udp.rs
mio-0.6.22/src/sys/unix/uds.rs
mio-0.6.22/src/sys/unix/uio.rs
mio-0.6.22/src/sys/windows/awakener.rs
mio-0.6.22/src/sys/windows/buffer_pool.rs
mio-0.6.22/src/sys/windows/from_raw_arc.rs
mio-0.6.22/src/sys/windows/mod.rs
mio-0.6.22/src/sys/windows/selector.rs
mio-0.6.22/src/sys/windows/tcp.rs
mio-0.6.22/src/sys/windows/udp.rs
mio-0.6.22/src/timer.rs
mio-0.6.22/src/token.rs
mio-0.6.22/src/udp.rs
mio-0.6.22/.cargo_vcs_info.json
```

Comment (pfmooney) [reply Cargo.toml:24]: That's using `cargo 1.42.0 (86334295e 2020-01-31)`

Comment (Thomasdezeeuw) [reply Cargo.toml:24]: 👍

**Comment by pfmooney** [comment]
Thanks for taking a look.  Requesting a 0.6.22 tag and push to crates.io, when it's convenient.

Cheers

**Comment by Thomasdezeeuw** [comment]
Released v0.6.22: https://crates.io/crates/mio/0.6.22.

**Comment by pfmooney** [comment]
> Released v0.6.22: https://crates.io/crates/mio/0.6.22.

Works great.  Thanks much!

**Pull request merged**